vortex-ui/src/main/java/mil/army/usace/hec/vortex/ui/GridToPointWizard.java

```diff
@@ -417,6 +521,7 @@ private void gridToPointTask() {
         GridToPointConverter converter = GridToPointConverter.builder()
                 .pathToGrids(pathToSource)
                 .variables(sourceGrids)
+                .statistics(statisticsList)
```

Comment:
Instead of passing a list of strings here I would pass the statistics as args in the `writeOptions` map. Check the map of args when you go to write in `DssDataWriter`.

Reply:
Addressed.